src/index.js

```diff
@@ -391,6 +391,15 @@ class SentryCliPlugin {
 
         return this.cli.releases.new(release);
       })
+      .then(() => {
```

Comment:
It has to be performed _after_ they are uploaded, otherwise there would be nothing to upload :P